index.html

```diff
@@ -73,124 +87,158 @@ <h2 class="proj-title">Tonic</h2>
               </div>
               <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.
               </p>
+              <p class="p-descrip4">
+                Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+              </p>
               <ul class="proj-lan">
                 <li class="lan">HTML</li>
+                <li class="lan">Ruby</li>
                 <li class="lan"> CSS</li>
                 <li class="lan">Javascript</li>
               </ul>
               <div class="btn-div">
                 <button class="btn" type="submit"> See Project</button>
               </div>
+          </div>
+            
             </div>
-            <div class="second-sec-div">
+            <div class="second-sec-div4">
               <div>
-                <img class="proj-img" src="images/Snapshoot Portfolio (3).png" alt="picture">
+                <img class="proj-img" src="images//mobile/Snapshoot Portfolio (3).png " alt="picture">
+                <img class="proj-img4" src="images//mobile/Snapshoot Portfolio (1).png " alt="picture">
               </div>
+              <div class="B">
               <div>
-                <h6 class="proj-title">Multi-Post <br>
-                  Stories</h6>
+                <h6 class="proj-title">Multi-Post <br> Stories</h6>
+                <h6 class="proj-title4">Uber Navigation</h6>
               </div>
               <div class="proj-intro">
                 <span class="intro-1">CANOPY</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images/mobile/Counter.png" alt="dot">
                 <span class="intro-2">Back end Dev</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images//mobile/Counter.png" alt="dot">
                 <span class="intro-3">2015</span>
               </div>
               <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <p class="p-descrip4">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+                
               </p>
               <ul class="proj-lan">
                 <li class="lan">HTML</li>
+                <li class="lan">Ruby on Rails</li>
                 <li class="lan"> CSS</li>
                 <li class="lan">Javascript</li>
               </ul>
               <div class="btn-div">
                 <button class="btn" type="submit"> See Project</button>
               </div>
             </div>
+            </div>
           </div>
         </section>
-        <section class="sec-3">
-          <div class="section3-div">
+          <section>
+             <div class="section-3">
             <div class="third-section">
```

Comment:
- [ ] Please, kindly don't be discouraged! As pointed earlier, kindly implement the About Section according to the given template. 👍 Reproducing perfectly a given design shows how well we pay attention to details as developers; and I'm sure recruiters are always on look out for such. ℹ️

- [optional] It would be great if you can fix one of the social media broken link. ℹ️  

1. `Let's connect` and `Get my Resume` elements are not well aligned.
2. Kindly increase the margin between the `social media icons` and `Get my Resume element`. 👍 

# Your design
<img width="369" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/230745815-72bf23a3-ddc6-49e1-b537-f2f57a48a940.png">

# Figma Design
<img width="402" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/230745876-932883a8-e656-455b-9de1-ba642945baed.png">